test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
```

Comment:
Please use a more descriptive email addresses, for example `old@c.com` and `new@c.com`.

Also we usually use `@example.com` as the domain name. `example.com` domain was designed exactly for that.